googleapiclient/discovery.py

```diff
@@ -190,7 +190,15 @@ def build(serviceName,
 
   requested_url = uritemplate.expand(discoveryServiceUrl, params)
 
-  content = _retrieve_discovery_doc(requested_url, http, cache_discovery, cache)
+  try:
+    content = _retrieve_discovery_doc(requested_url, http, cache_discovery,
+                                      cache)
+  except HttpError as e:
+    if e.resp.status == 404:
```

Comment:
Done, httplib is only available with python2, so I needed a hack for py3 though.